examples/cfd/lid_driven_cavity_2d.py

```diff
@@ -112,7 +112,7 @@ def post_process(self, i):
     precision_policy = PrecisionPolicy.FP32FP32
 
     velocity_set = xlb.velocity_set.D2Q9(precision_policy=precision_policy, backend=backend)
-    omega = 1.6
+    omega = 1.99
 
     simulation = LidDrivenCavity2D(omega, grid_shape, velocity_set, backend, precision_policy)
-    simulation.run(num_steps=5000, post_process_interval=1000)
+    simulation.run(num_steps=50000, post_process_interval=1000)
```

Comment:
Please keep this example the same as the original as lid-driven cavity example is our entry level example. We have KBC in windtunnel.

Reply:
I changed it back but added proper problem definition based on Re with sufficient number of iterations.